castle/cms/browser/controlpanel/status.py

```diff
@@ -5,51 +5,64 @@
 import celery.bin.base
 import celery.bin.celery
 import celery.platforms
+import os
 
 
 class StatusView(BrowserView):
 
     def docsplit(self):
         retval = self.DocsplitSearch()
         if retval[0]:
-            status = 'OK'
-            icon = '<span class="glyphicon glyphicon-ok" aria-hidden="true"></span> Docsplit: '
-            return '<span>' + icon + status + '</span>'
+            return True
         if not retval[0]:
-            status = 'ERROR: ' + retval[1]
-            icon = '<span class="glyphicon glyphicon-remove" aria-hidden="true"></span> Docsplit: '
-            return '<span>' + icon + status + '</span>'
+            return False
+
+    def docsplitError(self):
+        retval2 = self.DocsplitSearch()
+        if not retval2[0]:
+            return str(retval2[1])
 
     def elasticsearch(self):
         rs = redis.Redis("localhost")
+        rs2 = os.environ.get('REDIS_SERVER')
         try:
             rs.get(None)
-            status = 'OK'
-            icon = '<span class="glyphicon glyphicon-ok" aria-hidden="true"></span> Redis: '
+
+            return True
         except redis.ConnectionError as e:
-            status = 'ERROR: ' + str(e)
-            icon = '<span class="glyphicon glyphicon-remove" aria-hidden="true"></span> Redis: '
-        return '<span>' + icon + status + '</span>'
+            return False
 
     def celery(self):
+        retval = self.CeleryChecker()
+        if retval[0]:
+            return True
+        if not retval[0]:
+            return False
+
+    def celeryError(self):
+        retval2 = self.CeleryChecker()
+        if 'node' in retval2[1]:
+            return retval2[1]
+        else:
+            return 'Cannot connect to Redis'
```

Comment:
I think you mean 'Celery'

Reply:
Celery needs Redis in order to work so that message is what you get when Redis is turned off. I think I should say 'Celery cannot connect to Redis' for clarity. 
<img width="612" alt="screen shot 2018-06-08 at 11 28 07 am" src="https://user-images.githubusercontent.com/37121148/41169587-16cead36-6b0f-11e8-873a-41e9b4d110b6.png">

Reply:
Ah, got it!